pyproject.toml

```diff
@@ -60,7 +60,7 @@ itde = 'exasol_integration_test_docker_environment.main:main'
 [[tool.mypy.overrides]]
 module = [ "luigi.*", "docker.*", "humanfriendly", "configobj", "toml", "netaddr", "joblib.testing", "networkx",
             "fabric", "requests", "pyexasol", "paramiko.ssh_exception", "six", "jsonpickle", "exasol.toolbox.*",
-            "paramiko", "exasol"]
+            "paramiko", "exasol", "networkx.classes"]
```

Comment:
Do we still need the exasol override, if yes then the question which repo is the offender

Reply:
It's https://github.com/exasol/error-reporting-python

Reply:
Then we should fix it

Reply:
=> See https://github.com/exasol/error-reporting-python/pull/58